jvb/src/main/java/org/jitsi/videobridge/Videobridge.java

```diff
@@ -1131,13 +1131,17 @@ public IQ healthCheckIqReceived(@NotNull HealthCheckIQ iq)
          * The total number of times the dominant speaker in any conference
          * changed.
          */
-        public LongAdder totalDominantSpeakerChanges = new LongAdder();
+        public CounterMetric dominantSpeakerChanges = VideobridgeMetricsContainer.getInstance().registerCounter(
+                "dominant_speaker_changes",
+                "Number of times the dominant speaker in any conference changed.");
 
         /**
          * Number of endpoints whose ICE connection was established, but DTLS
          * wasn't (at the time of expiration).
          */
-        public AtomicInteger dtlsFailedEndpoints = new AtomicInteger();
+        public CounterMetric dtlsFailedEndpoints = VideobridgeMetricsContainer.getInstance().registerCounter(
+                "dtls_failed_endpoints",
```

Comment:
Can you rename this to endpoints_dtls_failed for consistency (we have other endpoints_ metrics, right?)

Reply:
Will do (and yes, we do)